libs/agno/tests/unit/knowledge/chunking/test_semantic_chunking.py

```diff
@@ -1,11 +1,20 @@
-import sys
-from types import ModuleType, SimpleNamespace
+"""Tests for SemanticChunking wrapper that adapts Agno embedders to chonkie."""
+
+from types import SimpleNamespace
+from unittest.mock import patch
+
+import pytest
+
+# Skip all tests if chonkie is not installed
+pytest.importorskip("chonkie")
```

Comment:
I'd rather have them break if we are missing the dep. It's there already for the CI tho so we are good